go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/bitly/go-simplejson v0.5.0 // indirect
 	github.com/deckarep/golang-set v1.7.1
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/droptheplot/abcgo v0.0.0-20171120220436-23529565504c // indirect
```

Comment:
is it needed for the service?

Reply:
It was updated automatically, so if you want I can remove this files.

Reply:
it was updated by running `abcgo.sh`, but not by running `go build`. I'd vote to checkout these two files.